app/src/main/java/com/example/neighborly/RequestActivity.java

```diff
@@ -90,10 +94,12 @@ public MessageModel parseSnapshot(DataSnapshot dataSnapshot) {
                 if (message != null) {
                     message.setMessageId(dataSnapshot.getKey());
                 }
-                String senderId = message.getSender().getId();
-                if (senderId != null && (lastUserToAnswer == null || !senderId.equals(curUser.getId()))){
-                    lastUserToAnswer = senderId;
+
+                UserModel user = message.getSender();
+                if (user != null && !user.getId().equals(curUser.getId())) {
```

Comment:
What happens if the same user comments more than once?
Maybe change the list to a set?

Reply:
done